### DISCUSSION TITLE
slider length always based on SV 1

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21046] [Q&A] [answered]
when you try to snap a slider end to a time notch, it bases it off of it was SV 1 not whatever SV it actually is. 
https://user-images.githubusercontent.com/78441708/199162922-5394f3f4-f7e1-43f7-998b-be6ef2407877.mp4

**Comment by smoogipoo** [comment]
I'm not sure what the expectation is, intuitively it's working as I'd expect - the slider path is preserved but its duration changes so the timeline has to change to reflect that.

Are you expecting it to do the opposite - lengthen/shorten the path but keep the timeline size the same? If so, I don't think that's easy/feasible to do (for shortening maybe, for lengthening probably not).

Perhaps I'm misunderstanding your intentions though.

**Reply by peppy** [reply]
it should be snapped to time value snaps. In the video it isn't (leaving the end of the slider completely out of time with the beatmap).

Likely a regression this release.

**Reply by smoogipoo** [reply]
Oh, I missed that... Thanks

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/21047
